src/tablecloth/api/columns.clj

```diff
@@ -104,35 +104,53 @@
        (f column strategy)
        (ds/add-or-update-column ds column-name)))
 
-(declare add-or-replace-column)
+(declare add-column)
 
-(defn- prepare-add-or-replace-column-fn
+(defn- prepare-add-column-fn
   [column-name column size-strategy]
   (cond
     (col/is-column? column) #(fix-column-size fix-column-size-column % column-name column size-strategy)
-    (dtype/reader? column) (prepare-add-or-replace-column-fn column-name (col/new-column column-name column) size-strategy)
+    (dtype/reader? column) (prepare-add-column-fn column-name (col/new-column column-name column) size-strategy)
     (iterable-sequence? column) #(fix-column-size fix-column-size-seq % column-name column size-strategy)
-    (fn? column) #(add-or-replace-column % column-name (column %) size-strategy)
+    (fn? column) #(add-column % column-name (column %) size-strategy)
     :else #(ds/add-or-update-column % column-name (dtype/const-reader column (ds/row-count %)))))
 
-(defn add-or-replace-column
+(defn ^:deprecated add-or-replace-column
```

Comment:
Please move `add-or-replace-column(s)` to a `tablecloth.api` and use `def` to refer to `add-column(s)`. Check how it's done with `write-csv!` function). Keeping two the same implementations (even if one is deprecated) can lead to some maintenance problems in the future.